src/dash/DashAdapter.js

```diff
@@ -457,6 +458,7 @@ function DashAdapter() {
             const id = eventBox.id;
             const messageData = eventBox.message_data;
 
+            event.type = schemeIdUri;
```

Comment:
I don't see `type` being used for inline events neither. Since `type` is not an attribute in `<xs:complexType name="EventType">` I suggest we remove `type`. The `schemeIdUri` can be accessed via `event.eventStream.schemeIdUri`.